android/Module.md

```diff
@@ -0,0 +1,4 @@
+# Module bdk-android
+The [bitcoindevkit](https://bitcoindevkit.org/) language bindings library for Android. Current version: `0.5.1`.
```

Comment:
Is it customary to put the latest version in this file? If not I'd rather leave it out so we only need to bump versions in the `build.gradle.kts` file.

```suggestion
The [bitcoindevkit](https://bitcoindevkit.org/) language bindings library for Android.
```

Reply:
No I don't think it is! But I haven't found how to display the version so far. So the docs are ambiguous as to which version they document... I'm sure there is a simple way but for some reason I couldn't find it.

Reply:
Done.